csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -38,9 +77,29 @@ public static string GetOrReadArgument(int index, string readMessage, params str
             }
         }
 
+        /// <summary>
+        /// <para>Output a input string to the console</para>
+        /// <para>Выводит переданныую строку</para>
+        /// </summary>
+        /// <param name="string">
+        /// <para>Text passed to the console</para>
+        /// <para>Текст переданный в консоль</para>
+        /// </param>
         [Conditional("DEBUG")]
         public static void Debug(string @string) => Console.WriteLine(@string);
 
+        /// <summary>
+        ///	<para>Output composit formatted text in the console</para>
+        /// <para>Выводит композитный форматированный текст в консоли</para>
```

Comment:
Take a look at Russian version of [the documentation of `Console.WriteLine`](https://docs.microsoft.com/ru-ru/dotnet/api/system.console.writeline?view=net-5.0#System_Console_WriteLine_System_String_System_Object___). And use it to combine with an example from the above.